src/Compilers/Core/MSBuildTaskTests/GenerateMSBuildEditorConfigTests.cs

```diff
@@ -97,6 +97,35 @@ public void MultipleItemMetaDataCreatesSections()
 ", result);
         }
 
+        [Fact]
+        [WorkItem(52469, "https://github.com/dotnet/roslyn/issues/52469")]
+        public void MultipleSpecialCharacterItemMetaDataCreatesSections()
```

Comment:
>MultipleSpecialCharacterItemMetaDataCreatesSections [](http://example.com/codeflow?start=20&length=51)

Please consider testing all the special characters from `EncodeString()` if not already tested.

Reply:
I've updated the tests to validate the full set of special chars.